ios/Podfile

```diff
@@ -17,11 +17,12 @@ target 'PassCulture' do
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable these next few lines.
-  use_flipper!
+  use_flipper!({ 'Flipper' => '0.74.0' })
```

Comment:
Erreur lors du build iOS:
 
Référence: https://stackoverflow.com/questions/66019068/event2-event-config-h-file-not-found